Integrations/ProofpointTAP_v2/ProofpointTAP_v2.py

```diff
@@ -189,9 +194,8 @@ def fetch_incidents(client, last_run, first_fetch_time, event_type_filter, threa
 
             threat_info_map = raw_event.get("threatsInfoMap", [])
             for threat in threat_info_map:
-                if threat["threatTime"] > last_fetch:
-                    last_event_fetch = last_event_fetch if last_event_fetch > threat["threatTime"] else threat[
-                        "threatTime"]
+                last_event_fetch = last_event_fetch if last_event_fetch > threat["threatTime"] else threat[
```

Comment:
you dont need to save the `last_event_fetch` anymore. Just save the `end_query_time`